opentelekomcloud/resource_opentelekomcloud_lb_listener_v2.go

```diff
@@ -193,20 +227,23 @@ func resourceListenerV2Read(d *schema.ResourceData, meta interface{}) error {
 
 	log.Printf("[DEBUG] Retrieved listener %s: %#v", d.Id(), listener)
 
-	d.Set("id", listener.ID)
-	d.Set("name", listener.Name)
+	d.Set("region", GetRegion(d, config))
```

Comment:
Please add a `mutlierror` to make sure no fields are set with errors. This is not common approach, but if possible, this should be done. Especially, for new fields.
e.g:
```go
mErr := multierror.Append(nil, 
	d.Set("region", GetRegion(d, config))
...
return mErr.ErrorOrNil()
```

Reply:
This is legacy code, but I can cange accordingly if neccessary

Reply:
That would be very nice of you :)